package exception;

import java.util.Scanner;

public class Ex01 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int num;
		
		try {
			System.out.print("정수 입력: ");
			num = sc.nextInt();
			
			System.out.println("입력 받은 값: " + num);
		}
		catch(Exception e) {
			e.printStackTrace();
		}
		System.out.println("Scanner 자원 해제~");
		sc.close();
	}

}
